apps/pdc-frontend/gql/graphql.ts

```diff
@@ -260,14 +260,12 @@ export type ComponentComponentsEForm = {
 
 export type ComponentComponentsEFormFiltersInput = {
   and?: InputMaybe<Array<InputMaybe<ComponentComponentsEFormFiltersInput>>>;
-  eFormURL?: InputMaybe<ComponentComponentsUrlFiltersInput>;
```

Comment:
Heeft dit iets met een focus issue te maken? Er lijkt in deze PR veel ongerelateerde code te zitten.

Reply:
Nee, ik heb deze wijzigingen uit deze PR gehaald. Ik heb een ander PR aangemaakt voor deze wijzigingen